ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -651,21 +885,61 @@ gst_tensor_src_tizensensor_stop (GstBaseSrc * src)
 static gboolean
 gst_tensor_src_tizensensor_event (GstBaseSrc * src, GstEvent * event)
 {
-  /** @todo NYI */
-
   /** No events to be handled yet */
   return GST_BASE_SRC_CLASS (parent_class)->event (src, event);
 }
 
+/**
+ * @brief Get possible GstCap from the configuration of self.
+ */
+static GstCaps *
+_ts_get_gstcaps_from_conf (GstTensorSrcTIZENSENSOR *self)
+{
+  TizenSensorSpec *spec;
+  gchar *tensor;
+  GstCaps *retval;
+
+  spec = g_hash_table_lookup (tizensensors, self->type);
+
+  if (FALSE == self->configured || SENSOR_ALL == self->type || NULL == spec) {
+    tensor = g_strdup_printf("other/tensor; other/tensors, num_tensors=1");
+  } else {
+    tensor = g_strdup_printf("other/tensor, dimension=%u:%u:%u:%u ; "
```

Comment:
maybe need `type=xx`
recommended to use gst_tensor_caps_from_config(), gst_tensors_caps_from_config()

Reply:
Although we cannot use ```gst_tensor(s)_caps_from_config``` here because we need both at this stage, adding type was done with https://github.com/nnsuite/nnstreamer/pull/1902/files#diff-907b175410f77e56b900fc43d5ef463dR942-R947